lib/type.g

```diff
@@ -190,26 +190,13 @@ end );
 ##  see&nbsp;<Ref Sect="Component Objects"/>
 ##  and&nbsp;<Ref Sect="Positional Objects"/> for the details.
 ##  <P/>
-##  The third argument <A>slots</A> is a list either of integers or of
-##  strings.
-##  In the former case,
-##  <A>rep</A> must be <Ref Filt="IsPositionalObjectRep"/> or a
-##  subrepresentation of it, and <A>slots</A> tells what positions of the
-##  objects in the representation <A>rep</A> may be bound.
-##  In the latter case,
-##  <A>rep</A> must be <Ref Filt="IsComponentObjectRep"/> or a
-##  subrepresentation of, and <A>slots</A> lists the admissible names of
-##  components that objects in the representation <A>rep</A> may have.
-##  The admissible positions resp. component names of <A>super</A> need not
-##  be listed in <A>slots</A>.
+##  The optional third and fourth arguments <A>slots</A> and <A>req</A>
+##  are unused (and never were) only provided for backwards compatibility.
```

Comment:
perhaps "are (and always were) unused and only provided for backwards compatibility"

Reply:
Indeed, that's much better. Fixed now